PixaPicker/PixaFavoritesViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  PixaFavoritesViewController.swift
+//  PixaPicker
+//
+//  Created by Thomas Caraway on 2/3/20.
+//  Copyright © 2020 Thomas Caraway. All rights reserved.
+//
+
+import Foundation
+import SDWebImage
+import UIKit
+import CoreData
+
+
+//NOTE: WHEN REMOVING PICS FROM FAVES, WILL ALSO REMOVE ANY DUPLICATES. WONT BE ISSUE ONCE WE DISABLE FAVORITE BUTTON AFTER IT GETS PRESSED IN SEARCHVIEWCONTROLLER
+class PixaFavoritesViewController: UIViewController, PixaSaveButtonDelegate, PixaFavoriteButtonDelegate, UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout, PixaFavoritesCoordinatorDelegate{
+    
+    @IBOutlet weak var imageCollectionView: UICollectionView!
+    let reuseID = "cell"
+    let favoritesCoordinator = PixaFavoritesCoordinator()
+    var delegate: PixaViewControllerDelegate?
```

Comment:
This should be flipped.  a `PixaFavoriteCoordinator` should be managing the `PixaFavoritesViewController` instead of this way.

Take a gander at this helpful post on the Coordinator pattern: 
https://www.hackingwithswift.com/articles/71/how-to-use-the-coordinator-pattern-in-ios-apps